packages/storybook-addon/package.json

```diff
@@ -19,13 +19,17 @@
     "@storybook/node-logger": "7.4.6",
     "webpack": "5.88.2",
     "webpack-virtual-modules": "0.6.1",
-    "@module-federation/utilities": "3.0.0"
+    "@module-federation/utilities": "3.0.0",
+    "@nx/react": "^17.1.3",
+    "@nx/webpack": "^17.1.3"
   },
   "peerDependencies": {
     "@storybook/core-common": "^6.5.16 || ^7.0.0",
     "@storybook/node-logger": "^6.5.16 || ^7.0.0",
     "webpack": "^5.75.0",
     "webpack-virtual-modules": "^0.5.0 || ^0.6.0",
-    "@module-federation/utilities": "3.0.0"
+    "@module-federation/utilities": "3.0.0",
```

Comment:
utils is repeated in here and dev deps. is that correct?

Reply:
yes